File: beginner/pagina01/1008/1008.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
	public static void main(String[] args) throws IOException {
		InputStreamReader ir = new InputStreamReader(System.in);
		BufferedReader in = new BufferedReader(ir); 

		Integer number = Integer.parseInt(in.readLine());
		Integer worked_hours = Integer.parseInt(in.readLine());
		double value_per_hour = Double.parseDouble(in.readLine());

		System.out.printf("NUMBER = %d\n", number);
		System.out.printf("SALARY = U$ %.2f\n", worked_hours * value_per_hour);
	}
}

File: beginner/pagina01/1008/1008.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {

	int employee_number, worked_hours;
	double received_per_hour, salary;

	cin >> employee_number >> worked_hours >> received_per_hour;

	salary = received_per_hour * worked_hours;

	cout << "NUMBER = " << employee_number << endl;
	cout << "SALARY = U$ " << fixed << setprecision(2) << salary << endl;

	return 0;
}